Data/ByteString/Builder/Prim/Internal/UncheckedShifts.hs

```diff
@@ -13,80 +10,16 @@
 -- Maintainer  : Simon Meier <iridcode@gmail.com>
 -- Portability : GHC
 --
--- Utilty module defining unchecked shifts.
---
--- These functions are undefined when the amount being shifted by is
--- greater than the size in bits of a machine Int#.-
---
-#if !defined(__HADDOCK__)
-#include "MachDeps.h"
-#endif
 
 module Data.ByteString.Builder.Prim.Internal.UncheckedShifts (
```

Comment:
It's a bit awkward to keep this module just for `caseWordSize_32_64`. How about moving `caseWordSize_32_64` to `D.B.Builder.Prim.Internal`?

Reply:
Done.